spec/requests/v1/term_pagination_spec.rb

```diff
@@ -24,6 +24,13 @@
     )).excluding('terms')
   end
 
+  it 'sets per_page to default when greater than 1000' do
```

Comment:
Can we have it set per_page to `URIService::MAX_PER_PAGE` when greater than URIService::MAX_PER_PAGE?

And could you change the text string to:
`it 'sets the per_page to URIService::MAX_PER_PAGE when greater than URIService::MAX_PER_PAGE' do`?

Reply:
I think we should follow what kaminari does, which it might do the behavior you described. I'll look into it and get back to you.

Reply:
Oh right, haha.  I forgot about that.  Then following kaminari works for me!

Reply:
Cool. I just double checked and kaminari works as expected.